enhancements/dns/externaldns-operator.md

```diff
@@ -235,15 +233,19 @@ such as a secret or configmap in an arbitrary namespace.
 By default, the operator would assume the cluster's platform is the desired DNS Provider. The operator would allow a user to specify
 an alternative External DNS Provider, such as BlueCat DNS. The ExternalDNS operator would be able to create multiple ExternalDNS deployments
 to run in concert, one for each ExternalDNS operator Custom Resource (similar to how the OpenShift [Ingress Operator](https://github.com/openshift/cluster-ingress-operator)
-consumes one or more Ingress Controller resources). The `domains` field would be used to filter source resources according to the requested DNS hostname
-(sans any domains matching any values specified in the `excludeDomains` field).
-If an ExternalDNS operator Custom Resource specifies multiple DNS Zones, multiple instances of ExternalDNS would be deployed within the same deployment accordingly.
+consumes one or more Ingress Controller resources). The `domains` field would be used to filter source resources based on the source resources desired hostname.
+The `domains` field should be expressive enough to allow a user to specify multiple sets of domains that are considered included or excluded.
+In the ExternalDNS operator CR above, ExternalDNS is configured to create DNS records with hostnames that fit under `my-cluster.my-domain.com` that are not under
+`apps.my-cluster.my-domain.con`.
+
+By default, ExternalDNS would create DNS records for all instances of the given Source resource. A union type would be defined to allow for
+source-specific parameters. For the example ExternalDNS CR above, the ExternalDNS operator would create DNS records for Route resources that have the annotation
+`some.annotation.io/foo=bar`, and would ignore the ExternalDNS custom Hostname Annotation (see [Implementation Details](#implementation-details) for more details about ExternalDNS
+annotations).
```

Comment:
The example has `hostnameAnnotationPolicy: Allow`; doesn't that mean the annotation would *not* be ignored?
```suggestion
source-specific parameters. For the example ExternalDNS CR above, the ExternalDNS operator would configure ExternalDNS to create DNS records for Route resources that have the annotation
`some.annotation.io/foo=bar`, and would allow the ExternalDNS custom Hostname Annotation to be used to specify a custom hostname (see [Implementation Details](#implementation-details) for more details about ExternalDNS
annotations).
```

Reply:
whoops! fixed. thanks!